### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by gracieflores**
This is one I started working on in late summer and I finally figured it out when working on the django upgrades. We are replacing travis with github actions.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

**Review by somexpert — APPROVED** [review]
Looks good and the tests seem to be running as expected with github actions :thumbsup:

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  container-job:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start the app
+        run: docker-compose up -d app
+      - name: Run the tests
+        run: docker-compose run --rm test --create-db
```
Comment (ldko) [inline .github/workflows/test.yml:17]: Did we lose the flake8 checking?

File: README.md [hunk 2]
```diff
@@ -59,6 +59,21 @@ $ docker-compose build
...
+
+Similar to docker and docker-compose, you will need to install, clone the repository and create a `secrets.json`.
+
+[Install or Enable Podman](https://podman.io/getting-started/installation).
+
+[Install Podman Compose](https://github.com/containers/podman-compose).
+
+If you have SELinux, you may need to temporarily add `:Z` to the base volumes in the docker-compose.yml. It will look like `.:/app/:Z`. You may also need to use `sudo` for your podman-compose commands.
+
+The rest of the steps are also similar. You will want to replace the word docker with podman. You will need to add one step before migrating:
```
Comment (ldko) [inline README.md:72]:
```suggestion
The rest of the steps are also similar. You will want to replace the word "docker" with "podman". You will need to add one step before migrating:
```

File: README.md [hunk 3]
```diff
@@ -59,6 +59,21 @@ $ docker-compose build
...
+```sh
+$ podman-compose up -d
```
[the remark this answers is not in the document]
Comment (gracieflores) [reply README.md:74]: Yes, this is in addition. Technically you can skip "warming up the db" for the podman steps, but if you don't run `podman-compose up -d` then it won't build your images.

**Review by ldko — APPROVED** [review]
Thanks for answering my questions. Other than the quotes (or use back ticks) comment I don't have anything else. :+1:

**Pull request merged**